test/asm/break.asm

```diff
@@ -0,0 +1,22 @@
+FOR V, 1, 100
+      PRINTLN "- {d:V}"
+      IF V == 5
+          PRINTLN "stop"
+          BREAK
+      ENDC
+      PRINTLN "cont"
+ENDR
+WARN "done {d:V}"
+rept 1
+  break
+  ; skips invalid code
+  !@#$%
+elif: macro
+  invalid
+endr
+warn "OK"
+rept 1
+  if 1
+    break
+  println "no endc"
```

Comment:
To make the check slightly more robust, I'd prefer bare `no endc` on that line, to ensure that they aren't counted even if part of a string, and would lex as a keyword if not for being a macro arg.